### ISSUE TITLE
Can not set alternateIdentifiers via API using "alternateIdentifiers" attribute but is returned in response

### ISSUE THREAD

**Issue by richardhallett** [datacite/lupo#674]
While the API docs do not say to use "alternateIdentifiers" attribute but just "identifiers" attribute, a valid metadata document that contains alternateIdentifiers does return this in the response payload.
According to the JSONAPI spec, this should mean you should be able to send back the response your eceived to update those attributes.

However sending alternateIdentifiers in the payload does not update the alternateIdentifiers.

## Steps to reproduce

1. Create JSON payload

```
{
  "data": {
    "type": "dois",
    "attributes": {
      "prefix": "10.5072",
      "alternateIdentifiers":[{"alternateIdentifierType":"URL","alternateIdentifier":"http://example.com/123"}]
    }
  }
}
```

2. Send http POST to https://api.stage.datacite.org/dois

**Comment by richardhallett** [comment]
Potentially related https://github.com/datacite/lupo/issues/641

**Comment by richardhallett** [comment]
This has been fixed and was part of same issue as #641
